application/console/controllers/CronController.php

```diff
@@ -117,6 +117,13 @@ public function actionDeleteInactiveUsers()
         User::deleteInactiveUsers();
     }
 
+    public function actionSendInactiveUsersEmail()
```

Comment:
Add `actionSendInactiveUsersEmail` to `actionAll`. We recently change the cron implementation and no longer have the flexibility to choose different schedules for each cron action.